runtime/vara/src/lib.rs

```diff
@@ -340,16 +340,27 @@ impl pallet_preimage::Config for Runtime {
     type ByteDeposit = PreimageByteDeposit;
 }
 
+parameter_types! {
+    // For weight estimation, we assume that the most locks on an individual account will be 50.
+    // This number may need to be adjusted in the future if this assumption no longer holds true.
+    pub const MaxLocks: u32 = 50;
```

Comment:
We can decrease this amount since we don't use currency reservation logic

Reply:
This can be anything. Like staking, vesting, voting